pkg/virt-launcher/virtwrap/cmd-server/server.go

```diff
@@ -464,6 +464,22 @@ func (l *Launcher) Exec(ctx context.Context, request *cmdv1.ExecRequest) (*cmdv1
 	return resp, nil
 }
 
+func (l *Launcher) GuestPing(ctx context.Context, request *cmdv1.GuestPingRequest) (*cmdv1.GuestPingResponse, error) {
+	resp := &cmdv1.GuestPingResponse{
+		Response: &cmdv1.Response{
+			Success: true,
+		},
+	}
+	err := l.domainManager.GuestPing(request.DomainName)
+	if err != nil {
+		resp.Response.Success = false
+		resp.Response.Message = err.Error()
+		return resp, err
+	}
+	resp.ExitCode = int32(0)
```

Comment:
If it's always 0 we might just as well drop it from the proto and determine an exitcode in virt-probe that matches our success (0/1).

Reply:
Removed `exitCode` from proto now, `virt-probe` will exit with exit code simply depending on the error now